kyuubi-server/src/main/scala/org/apache/kyuubi/server/trino/api/TrinoContext.scala

```diff
@@ -187,13 +187,20 @@ object TrinoContext {
       queryHtmlUri: URI,
       queryStatus: OperationStatus,
       columns: Option[TGetResultSetMetadataResp] = None,
-      data: Option[TRowSet] = None): QueryResults = {
+      data: Option[TRowSet] = None,
+      updateType: String = null): QueryResults = {
```

Comment:
how about pass a functon how to convert row se to list ? `convertRowListFunc: => List[List[Object]]`